website/content/en/advocacy/_index.adoc

```diff
@@ -19,25 +17,23 @@ This page provides links to more information to help you do this.
 
 * link:../status[FreeBSD quarterly status reports]
 +
-Quarterly status reports detailing activity within and surrounding FreeBSD.
+Details of activities within and surrounding FreeBSD.
 
-* link:whyusefreebsd[Why Use FreeBSD?]
+* link:whyusefreebsd[Why use FreeBSD?]
 +
-Explanations given by existing users as to why FreeBSD should be used.
+Users explain why FreeBSD should be used.
 
-* link:myths[*BSD Myths]
+* link:myths[*BSD myths]
 +
-Describes and debunks some of the myths that surround the *BSD projects.
+Describes and debunks some of the mythology.
 
-* link:../press[FreeBSD in the Press]
+* link:../press[FreeBSD in the press]
 +
-Contains many links to articles that have appeared which mention FreeBSD.
+Links to articles.
 
 == Sites using FreeBSD
```

Comment:
This is not, or no longer, a fitting subtitle. I don't know how to change it _without_ breaking the `#_sites_using_freebsd` anchor.

Reply:
I'll try <https://docs.asciidoctor.org/asciidoc/latest/attributes/id/#inline-assignment> …

Reply:
Anchor survived: 

![image](https://user-images.githubusercontent.com/192271/152705508-5646bdff-c025-47f1-a7ff-3d48bd8dcc74.png)